docs: Add troubleshooting for slow Capella and unresponsive Xpra (#1750)
DSD-DBS/capella-collab-manager: docs/docs/user/tools/capella/troubleshooting/index.md

    - End your session and request a new session in the Capella Collaboration Manager.

    If none of these workarounds help, the only solution is to reset your workspace. Please contact your administrator for assistance.

<!-- prettier-ignore -->
??? question "My Capella is slow. What can I do? <a href="#slow-capella">:material-link:</a>"

    <span id="slow-capella" />

    A slow Capella instance can have multiple reasons. Here are some common issues:

    1.  __Disable the automatic refresh in the Semantic Browser__

        The automatic refresh of the Semantic Browser slows down Capella when connected to a TeamForCapella repository.

>         Per default, the Semantic Browser auto-refresh is enabled. You can disable it in each session with the button marked in the screenshot (red/right box).

Reviewer: ```suggestion
        By default, the Semantic Browser auto-refresh is enabled. You can disable it in each session using the button marked in the screenshot (red/right box).
```
